connect.go

```diff
@@ -201,6 +208,10 @@ func (c *Connection) runClient() error {
 				message = receiveMessage(connection)
 				m := strings.Split(message, "-")
 				encryptedData, salt, iv, sendersAddress := m[0], m[1], m[2], m[3]
+                                if sendersAddress == "0.0.0.0" {
+                                        notPresent = true

```

Comment:
add a time.Sleep(1 * time.Second) here to help thwart attackers who want to spam for connections

Reply:
Added.